### PR TITLE
Make displaying DICOM verification image optional

### PR DESCRIPTION
**Pull request by niwilso**
## Change Description

Minor but important update to prevent automatically generating and displaying the verification image each time  `DicomImagePiiVerifyEngine.eval_dicom_instance()` is called.

## Issue reference

This PR fixes issue #997 

## Checklist

- [x] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [ ] I have signed the CLA (if required)
- [x] My code includes unit tests
- [x] All unit tests and lint checks pass locally
- [ ] My PR contains documentation updates / additions if required

### CODE REVIEW

File: presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py [hunk 1]
```diff
@@ -45,12 +45,14 @@ def verify_dicom_instance(
         self,
         instance: pydicom.dataset.FileDataset,
         padding_width: int = 25,
+        display_image: bool = True,
         **kwargs,
     ) -> Tuple[PIL.Image.Image, dict, list]:
```
Comment (SharonHart) [inline presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py:50]: you can use the `Optional` type hint to make it clear to the client that there is a possibility for an image to not be returned

File: presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py [hunk 2]
```diff
@@ -45,12 +45,14 @@ def verify_dicom_instance(
...
         """Verify PII on a single DICOM instance.
 
         :param instance: Loaded DICOM instance including pixel data and metadata.
         :param padding_width: Padding width to use when running OCR.
+        :param display_image: If the verificationimage is displayed and returned.
```
Comment (SharonHart) [inline presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py:55]: displayed? is it displayed in the ImagePiiVerifyEngine?

File: presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py [hunk 3]
```diff
@@ -80,9 +82,12 @@ def verify_dicom_instance(
         )
 
         # Get image with verification boxes
-        verify_image = self.verify(
-            image, ad_hoc_recognizers=[deny_list_recognizer], **kwargs
-        )
+        if display_image:
+            verify_image = self.verify(
```
Comment (SharonHart) [inline presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py:86]: Optionally `verify_image = self.verify(..) if display_image else None`

File: presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py (hunk 2 above)
Comment (niwilso) [reply presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py:55]: If `self.verify` is called, then the image is displayed as well as returned.

**Review by SharonHart — DISMISSED** [review]

File: presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py (hunk 1 above)
Comment (SharonHart) [reply presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py:50]: @niwilso I mean the return type :)
Tuple[Optional[PIL.Image.Image], dict, list]

**Review by SharonHart — APPROVED** [review]

**Review by omri374 — APPROVED** [review]

**Pull request merged**